deploy/services-demo/create_test_user.sh

```diff
@@ -8,32 +8,40 @@ set -e
 # if you have direct access to brig
 #
 
-# Usage:
-#   --csv                      Output users in CSV format
-#   --count=INT                Generate several users (by default it's just one)
-
-CSV=false
-COUNT=1
-
-# Parse CLI options
-
-if ! [ $# -eq 0 ]; then
-    TEMP=`getopt -o "" -l csv,count: -n 'create_test_user.sh' -- "$@"`
-    eval set -- "$TEMP"
-
-    while true ; do
-        case "$1" in
-            --csv) CSV=true ; shift ;;
-            --count)
-                case "$2" in
-                    "") shift 2 ;;
-                    *) COUNT=$2 ; shift 2 ;;
-                esac ;;
-            --) shift ; break ;;
-            *) echo "Unrecognized option $1" ; exit 1 ;;
-        esac
-    done
-fi
+USAGE="USAGE: $0
+    -n <N>:  Create <N> users. default: 1
+    -h <host>: Base URI of brig. default: http://localhost:8082
+    -c: Output as headerless CSV in format 'User-Id,Email,Password'. default: false
+"
+
+BRIG_HOST="http://localhost:8082"
+COUNT="1"
+CSV="false"
+
+# Option parsing:
+# https://sookocheff.com/post/bash/parsing-bash-script-arguments-with-shopts/
```

Comment:
ah, so it *is* a solved problem in bash?  cool!  :)  gotta remember this script for the next time i need this!

Reply:
It's still not great; but it's better than nothing!

This is my old boss's blog; he has good stuff!